api.planx.uk/modules/send/bops/bops.ts

```diff
@@ -155,12 +289,17 @@ interface FindApplication {
  */
 async function checkBOPSAuditTable(
   sessionId: string,
+  version: string,
 ): Promise<Record<string, string>> {
+  const searchString = `%/api/${version}/planning_applications`;
```

Comment:
This is a good catch and quirk I hadn't anticipated!

I think once we're sending just one application type it would be nice to come back and remove this check also just to keep thing simple (even though it will still be valid code and work as expected).

Reply:
Yep fully agree :+1: